tests.yml

```
@@ -57,3 +57,7 @@ va/tests/behat:
   command: |
     cd tests/behat
     behat --colors
+
+va/tests/status-errors:
```

We should sync on naming here as I introduced two more here https://github.com/department-of-veterans-affairs/va.gov-cms/pull/3148/files: 

- va/tests/php-warning
- va/tests/php-error

Maybe:

```suggestion
va/tests/status-error:
```